src/schedule/ll_schedule.c

```diff
@@ -153,12 +153,12 @@ static int is_ll_pending(struct ll_schedule_data *queue)
 	return pending_count;
 }
 
-static inline void ll_next_timeout(struct ll_schedule_data *queue,
+static void ll_next_timeout(struct ll_schedule_data *queue,
 				   struct task *work)
```

Comment:
You may wish to fix this checkpatch issue too (alignment should match open parenthesis)

Reply:
done, thanks